cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -127,6 +128,12 @@
   List<RunRecord> getRuns(Id.Program id, ProgramRunStatus status, long startTime, long endTime, int limit);
 
   /**
+   * Fetches all active run records.
```

Comment:
What do mean by active? I guess you mean `ProgramRunStatus.RUNNING`, right? better be consistent with naming. Also better have this method more generic as:

`List<RunRecord> getRuns(ProgramRunStatus status)`.